File: VB/04_TableAndChart/SetTableStyle.vb
Imports Spire.Presentation
Imports System.ComponentModel
Imports System.Text

Namespace SetTableStyle
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'creat a ppt document and load file
			Dim ppt As New Presentation()
			ppt.LoadFromFile("..\..\..\..\..\..\Data\test.pptx")

			'get tbe table
			Dim table As ITable = TryCast(ppt.Slides(0).Shapes(0), ITable)

			'set the table style from TableStylePreset and apply it to selected table
			table.StylePreset = TableStylePreset.DarkStyle1Accent6

			'save the file
			ppt.SaveToFile("tableStyle.pptx", FileFormat.Pptx2010)
			Process.Start("tableStyle.pptx")
		End Sub
	End Class
End Namespace

File: CS/04_TableAndChart/SetTableStyle.cs
using Spire.Presentation;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SetTableStyle
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //creat a ppt document and load file
            Presentation ppt = new Presentation();
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\test.pptx");
            
            //get tbe table
            ITable table = ppt.Slides[0].Shapes[0] as ITable;

            //set the table style from TableStylePreset and apply it to selected table
            table.StylePreset = TableStylePreset.DarkStyle1Accent6;

            //save the file
            ppt.SaveToFile("tableStyle.pptx", FileFormat.Pptx2010);
            System.Diagnostics.Process.Start("tableStyle.pptx");
        }
    }
}
